### PR TITLE
chore: don't enable v1 for 1.x version

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

File: control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs [hunk 1]
```diff
@@ -24,13 +24,20 @@ impl crate::controller::io_engine::HostApi for super::RpcClient {
...
+        let inner = data.into_inner();
+
+        let registration_info = match inner.registration_info {
             Some(info) => info,
             None => {
                 // The dataplane did not send anything in registration info, which should
                 // not happen.
-                return Err(SvcError::NodeNotOnline {
-                    node: self.context.node().clone(),
+                return Err(SvcError::Unimplemented {
```
Comment (tiagolobocastro) [inline control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs:34]: can you upgrade the comment as well?

File: control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs [hunk 2]
```diff
@@ -24,13 +24,20 @@ impl crate::controller::io_engine::HostApi for super::RpcClient {
                 request: "v1::get_mayastor_info",
             })?;
 
-        let registration_info = match data.into_inner().registration_info {
+        let inner = data.into_inner();
+
+        let registration_info = match inner.registration_info {
```
Comment (tiagolobocastro) [inline control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs:29]: nit: preferred it as a single line :D

**Review by tiagolobocastro — APPROVED** [review]

File: control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs (hunk 2 above)
Comment (Abhinandan-Purkait) [reply control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs:29]: That was causing issue below in line 39, returning the version.

File: control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs (hunk 1 above)
Comment (Abhinandan-Purkait) [reply control-plane/agents/src/bin/core/controller/io_engine/v1/host.rs:34]: Sure.

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**